test/prop_statem.erl

```diff
@@ -198,8 +198,10 @@ did_open(Uri, LanguageId, Version, Text) ->
 did_open_args(_S) ->
   [els_proper_gen:uri(), <<"erlang">>, 0, els_proper_gen:tokens()].
 
-did_open_pre(#{connected := Connected} = _S) ->
-  Connected.
+did_open_pre(#{ connected := Connected
```

Comment:
The protocol establishes that the client cannot send any of these requests before the `initialized`, so preventing this from happening in the property-based tests. This is to avoid some flakyness due to the fact the indexing now happens after the `initialized` and these calls effectively by-pass the server forcing indexing of the file. If the DB is not ready when the `did_open` is sent, things would crash in the background.